definitions/tools/phlat.wdl

```diff
@@ -0,0 +1,55 @@
+version 1.0
+
+task phlat {
+  input {
+    String phlat_name = "phlat" 
+    File cram
+    File cram_crai
+    File reference
+    File reference_fai
+    Int nthreads = 8
+    Int mem = 20
+    String index_dir = "" # optional if indexes are within run.b38.sh default dir
+   }
+
+  Int space_needed_gb = 10 + round(5*size([cram, cram_crai, reference, reference_fai], "GB"))
+  runtime {
+    memory: "~{mem}GB"
+    cpu: nthreads
+    docker: "mgibio/phlat:1.1_withindex"
+    disks: "local-disk ~{space_needed_gb} HDD"
+    bootDiskSizeGb: 3*space_needed_gb
```

Comment:
Does the boot disk need to scale this severely with the size of the input data?

Reply:
I think I was mimicking optitype.wdl when I created phlat.wdl

If there is a better way to do this, I am all for it. Would times 2 be better?

Reply:
I'm working on a resource usage script that I hope will help us see how much disk space is actually being used so that we can fine tune some of these resource requests.